README.md

```diff
@@ -27,8 +27,8 @@ The [Atomic Developer Bundle](https://github.com/projectatomic/adb-atomic-develo
 
 ## Get Involved/Contact Us
 
-  IRC: #atomic and #nulecule on freenode
-  Mailing List: container-tools@redhat.com
+  * IRC: #atomic and #nulecule on freenode
+  * Mailing List: container-tools@redhat.com
```

Comment:
I will suggest a hyperlink for the mail i.e. [container-tools@redhat.com](https://www.redhat.com/mailman/listinfo/container-tools)